src/components/Header/Header.scss

```diff
@@ -0,0 +1,83 @@
+@import "../../styles/mixins";
+
+.header {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 46px;
+  @include desktop {
+    height: 64px;
+    gap: 32px;
+  }
+
+  .hamburger-menu {
+    position: absolute;
+    top: 11px;
+    left: 20px;
+    width: 24px;
+    height: 24px;
+    cursor: pointer;
+    @include desktop {
+      display: none;
+    }
+  }
+
+  .app-logo {
+    width: 33px;
+    height: 32px;
+    flex-shrink: 0;
+    cursor: pointer;
+    @include mobile {
+      position: absolute;
+      left: 50%;
+      transform: translateX(-50%);
+    }
+    @include desktop {
+      margin-left: 127px;
```

Comment:
why there is media for mobile and desktop, the mobile should be the default and only desktop media